### DISCUSSION TITLE
what's the output of  longest_iso.py

### DISCUSSION THREAD

**Discussion by smallfishcui** [hahnlab/CAFE5#134] [Tutorial] [answered]
Hi,

I was using  longest_iso.py to get the longest isoform for my annotation. However it just printed out the names of the files? What does that mean? Are those files per se already longest isoforms?

thanks,
Cui

**Comment by benfulton** [comment, marked as answer]
It prepends "longest_" to file names as it processes them. Then you can take all the files that start with "longest" and concatenate them in order to have your final input file. From the tutorial:

> In order to keep all but the longest isoforms, and place all sequences from all species into a single .fa file for the next tutorial step, run the following commands in your shell from the tutorial folder:

```
$ python longest_iso.py -d twelve_spp_proteins/
$ cat twelve_spp_proteins/longest*.fa > makeblastdb_input.fa
